inception/inception/BUILD

```diff
@@ -153,7 +153,7 @@ py_binary(
 
 sh_binary(
```

Comment:
Could you add a different target in the BUILD one for the current .sh and a new one for the .py